sys/include/net/ng_ndp/types.h

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ng_ndp_types    Types for IPv6 neighbor discovery
+ * @ingroup     net_ng_ndp
+ * @brief       IPv6 neighbor discovery message types
+ * @{
+ *
+ * @file
+ * @brief       IPv6 neighbor discovery message type definitions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NG_NDP_TYPES_H_
+#define NG_NDP_TYPES_H_
+
+#include <inttypes.h>
```

Comment:
`inttypes.h` provides the `PRI.[0-9]\+` macros, too. This is irrelevant here, but in the *.c-files is not… It's a case of clashing consistencies:

* `stdint.h` to be consistent with `stdio.h`, `stdlib.h` etc.
* `inttypes.h` to stay consistent with *.c-files utilizing the `PRI.[0-9]\+` macros for output…

I don't know what's better :-/